perllib/FixMyStreet/App/Controller/Admin/Users.pm

```diff
@@ -394,8 +394,10 @@ sub edit : Chained('user') : PathPart('') : Args(0) {
             id => $_->id,
             category => $_->category,
             active => $active_contacts{$_->id},
+            group => $_->{group} // $_->get_extra_metadata('group') // [''],
```

Comment:
I think `$_->get_extra_metadata('group')` will do here (and in Templates.pm the same), as the loop in stash_category_groups sorts out the default for you, and it's always an object here.